### PR TITLE
Add note about service CIDR bug to changes

### PR DESCRIPTION
**Pull request by saschagrunert**
Analogous change to https://github.com/kubernetes/kubernetes/pull/86759

### CODE REVIEW

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-18489--kubernetes-io-master-staging.netlify.com/docs/setup/release/notes/

File: content/en/docs/setup/release/notes.md [hunk 1]
```diff
@@ -83,6 +83,7 @@ The Kubernetes in-tree storage plugin to Container Storage Interface (CSI) migra
...
 - client-go discovery clients constructed using `NewDiscoveryClientForConfig` or `NewDiscoveryClientForConfigOrDie` default to rate limits that cause normal discovery request patterns to take several seconds. This is fixed in https://issue.k8s.io/86168 and will be resolved in v1.17.1. As a workaround, the `Burst` value can be adjusted higher in the rest.Config passed into `NewDiscoveryClientForConfig` or `NewDiscoveryClientForConfigOrDie`.
+- the IP allocator in v1.17.0 can return errors such as `the cluster IP <ip> for service <service-name> is not within the service CIDR <cidr>; please recreate` in the logs of the kube-apiserver. The cause is incorrect CIDR calculations if the service CIDR (`--service-cluster-ip-range`) is set to bits lower than `/16`. This is fixed in http://issue.k8s.io/86534 and will be resolved in v1.17.1.
```
Comment (kbhawkey) [inline content/en/docs/setup/release/notes.md]: @saschagrunert , this change looks good. One nit: Line 86, s/the/The/

Comment (saschagrunert) [reply content/en/docs/setup/release/notes.md]: Yep, fixed as suggested :+1:

**Pull request merged**